README.md

```diff
@@ -38,16 +38,21 @@ Enjoy your new RPM.
 
 ### Ubuntu/Debian
 
-Building a .deb should be straight forward with:
+On recent Debian/Ubuntu distributions you should be able to install with:
+```
+# apt-get install TODO
```

Comment:
@lhw I meant more like the nm ssh packages they should `apt-get install`.Probably `network-manager-ssh` and `network-manager-ssh-gnome`?